app/src/main/java/com/pvp/app/ui/screen/profile/ProfileScreen.kt

```diff
@@ -260,70 +264,75 @@ private fun ProfileHeader(
-                text = "Welcome, $userNameDisplay",
-                style = MaterialTheme.typography.titleLarge,
-                fontStyle = FontStyle.Italic,
-                fontSize = 30.sp
+                text = "Welcome,",
+                fontSize = 28.sp
             )
 
-            IconButtonWithDialog(
+            Row(
                 modifier = Modifier
-                    .padding(
-                        start = 5.dp,
-                        top = 4.dp
-                    ),
-                icon = Icons.Outlined.Edit,
-                iconSize = 30.dp,
-                iconDescription = "Edit Icon Button",
-                confirmButtonContent = {
-                    Text("Save")
-                },
-                dismissButtonContent = {
-                    Text("Cancel")
-                },
-                dialogTitle = {
-                    Text("Editing Username")
-                },
-                dialogContent = {
-                    OutlinedTextField(
-                        value = userNameEditing,
-                        onValueChange = { userNameEditing = it },
-                        label = { Text("Username") },
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(bottom = 16.dp)
-                    )
-                },
-                onConfirmClick = {
-                    userNameEditing = userNameEditing.trim()
+                    .underline(),
+            ) {
+                Text(
+                    text = userNameDisplay,
+                    fontSize = 28.sp
+                )
+
+
+                IconButtonWithDialog(
+                    modifier = Modifier
+                        .padding(
+                            start = 5.dp,
+                            top = 4.dp
+                        ),
+                    icon = Icons.Outlined.Edit,
+                    iconSize = 30.dp,
+                    iconDescription = "Edit Icon Button",
+                    confirmButtonContent = {
+                        Text("Save")
+                    },
+                    dismissButtonContent = {
+                        Text("Cancel")
+                    },
+                    dialogTitle = {
+                        Text("Editing Username")
+                    },
+                    dialogContent = {
+                        OutlinedTextField(
+                            value = userNameEditing,
+                            onValueChange = { userNameEditing = it },
+                            label = { Text("Username") },
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(bottom = 16.dp)
+                        )
+                    },
+                    onConfirmClick = {
+                        userNameEditing = userNameEditing.trim()
+                        val length = userNameEditing.length
```

Comment:
These are different purpose code blocks too. Variable assignation/Variable initialiation. Should separate.